packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -178,7 +155,7 @@ export const ApiExplorer: FC<ApiExplorerProps> = ({
     neededSpec = spec?.key
   }
 
-  return (
+  return isSynced ? (
```

Comment:
Adding this here to discuss with @bryans99 . I don't know if this check here is needed or how to go about using this synced variable, considering that without it I get the same behavior and tests passing.